src/components/Navbar/MenuItem/MenuItem.tsx

```diff
@@ -27,7 +28,9 @@ export const MenuItem = (props: MenuItemProps) => {
     <Menu.Item
       active={activePage === section}
       onClick={(e: React.MouseEvent) => {
-        isMobile ? onToggleShowSubMenu(e, true, section) : mainRedirect()
+        isMobile && !isExtraButton
```

Comment:
WDYT about checking if the `onToggleShowSubMenu` prop is defined instead of adding a new prop.
Both work for me anyway.

Reply:
Nevermind, your option is better. My suggestion won't work